Sources/AppCoinsSDK/Models/Services/Billing/AppCoinBillingClient.swift

```diff
@@ -53,13 +48,13 @@ internal class AppCoinBillingClient : AppCoinBillingService {
         }
     }
     
-    internal func getPaymentMethods(value: String, currency: Coin, result: @escaping (Result<GetPaymentMethodsRaw, BillingError>) -> Void) {
+    internal func getPaymentMethods(value: String, currency: String, result: @escaping (Result<GetPaymentMethodsRaw, BillingError>) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done